docs/source/kinesis.rst

```diff
@@ -0,0 +1,97 @@
+Kinesis Streams
+===============
+
+Overview
+--------
+
+* StreamAlert utilizes AWS Kinesis Streams for real-time data ingestion from `datasources <datasources.html>`_
+* A Kinesis Stream is created for each `cluster <clusters.html>`_ you define.
+* `Key concepts <https://docs.aws.amazon.com/streams/latest/dev/key-concepts.html>`_
+
+At a high-level:
+
+* A Kinesis Stream is created for each cluster you define
+* A Kinesis Stream is composed of one or more shards
+* The number of shards you choose is dependent on your data volume (see Concepts & Limits)
```

Comment:
I see `Limits` but where is `Concepts`? Is this a reference to the aws `key concepts` link above? Could we just link to these here too?